source/pidomus.cc

```diff
@@ -142,6 +142,12 @@ declare_parameters (ParameterHandler &prm)
                   "false",
                   Patterns::Bool());
 
+  add_parameter(  prm,
+                  &use_double_preconditioner,
+                  "Use double preconditioner",
```

Comment:
`double` is ambiguous. Maybe `Enable finer preconditioner`? Also add the documentation to this entry.